overlord/snapstate/snapstate_test.go

```diff
@@ -5929,8 +5957,12 @@ func (s *snapmgrTestSuite) TestInstallPathSkipConfigure(c *C) {
 	ts, err := snapstate.InstallPath(s.state, &snap.SideInfo{RealName: "some-snap", SnapID: "some-snap-id", Revision: snap.R(1)}, snapPath, "edge", snapstate.Flags{SkipConfigure: true})
 	c.Assert(err, IsNil)
 
-	runHook := taskWithKind(ts, "run-hook")
-	c.Assert(runHook, IsNil)
+	runHooks := findTasks(ts, func(task *state.Task) bool {
+		return task.Kind() == "run-hook"
+	})
+	// only one hook task for install, no configure hook
+	c.Assert(runHooks, HasLen, 1)
+	c.Assert(runHooks[0].Summary(), Equals, `Install hook of snap "some-snap"`)
```

Comment:
This doesn't look like the place to check for this. This test is about the checks below.